src/containers/entitysetsearch/EntitySetSearchResults.js

```diff
@@ -36,14 +39,40 @@ export default class EntitySetSearchResults extends React.Component {
   }
 
   onRowSelect = (selectedId, selectedRow, selectedEntitySetId, selectedPropertyTypes) => {
-    this.setState({ selectedId, selectedRow, selectedEntitySetId, selectedPropertyTypes });
+    EntityDataModelApi.getEntitySet(selectedEntitySetId)
```

Comment:
none of the entity set search stuff is set up with redux yet, so it made more sense to me for that to be all done at once. I can try to do a partial redux thing though if you think that's better